BootloaderCorePkg/Stage1B/Stage1B.c

```diff
@@ -744,8 +743,8 @@ ContinueFunc (
                     MbTmpAlgHash,
                     Stage1bParam->KeyHashManifestHash,
                     EV_EFI_PLATFORM_FIRMWARE_BLOB,
-                    sizeof(KeyHashFwBlob),
-                    (UINT8 *)&KeyHashFwBlob);
+                    (UINT32)KeyHashFwBlob.BlobLength,
```

Comment:
How about directly use KeyHashBlob->UsedLength and remove variable KeyHashFwBlob?

Reply:
done